src/components/shared/product-card.tsx

```
@@ -0,0 +1,122 @@
+"use client"
+
```

ESLint

[error] Missing semicolon.

(semi)